### ISSUE TITLE
$ITERM2_SQUELCH_MARK breaks ZSH Shell Integration

### ISSUE THREAD

**Issue by vaaralav** [gnachman/iterm2-website#63]
I just installed Shell Integration and got the following error:
`.iterm2_shell_integration.zsh:107: parse error near `]]'`

Removing `|| $ITERM2_SQUELCH_MARK` on that line fixed the issue.

**Comment by gnachman** [comment]
Strange, I cannot reproduce this. If you change it to this does it fix it?

```
if [[ $PS1 == *"$(iterm2_prompt_mark)"* ]] || [[ $ITERM2_SQUELCH_MARK ]]
```

**Comment by silvanet123** [comment]
I am having the same issue.  gnachman, I tried your request change and it did not fix it.  removing "|| $ITERM2_SQUELCH_MARK"  fixes it for me as well.

**Comment by gnachman** [comment]
What version of zsh exhibits this behavior?

Does this work better:
```
if [[ $X == *"$(iterm2_prompt_mark)"* ]] || (( ${+ITERM2_SQUELCH_MARK} )); then
```

**Comment by vaaralav** [comment]
```shell
> zsh --version
zsh 5.0.2 (i686-pc-linux-gnu)
```

**Comment by silvanet123** [comment]
#zsh --version
zsh 5.0.2 (x86_64-redhat-linux-gnu)

the new line gets past line 107 and I get the following error
.iterm2_shell_integration.zsh:111: parse error near `PS1="$PS1%{$(iterm2_...'

**Comment by gnachman** [comment]
Should be fixed by commit eb2beac. Please verify:

https://raw.githubusercontent.com/gnachman/iterm2-website/master/source/shell_integration/zsh

**Comment by silvanet123** [comment]
The error has gone away for me.  Thanks!!
